test/data/NarrativeTest.test_input_params.spec.json

```diff
@@ -98,7 +98,10 @@
                 "regex_constraint": [],
                 "valid_ws_types": [
                     "KBaseGenomes.Genome"
-                ]
+                ],
```

Comment:
I added this to test data so that at least one parameter has a dynamic_dropdown. It's not actually a valid app spec in some ways, but this seemed expedient for testing.